docs/source/tutorial/tutorial-execute-query.md

```diff
@@ -0,0 +1,124 @@
+
+![detail about launches query](images/launches_detail.png)
+
+In the right panel, you see both the query itself and information about what the query returns. You can use this information to write a query you'll eventually add to your app. 
+
+In the left-hand text area, add the following lines to start creating a query that will fetch a list of all available launches: 
+
+```graphql
+query LaunchList {
+}
+```
+
+The Apollo iOS SDK requires every query to have a name (even though this isn't required by the GraphQL spec). The query above has the name `LaunchList`.
+
+Next, between the query's curly braces, start typing `la`. An autocomplete box pops up and shows you options based on what's in the schema:
+
+![example of autocomplete](images/grapqhiql_autocomplete.png)
+
+GraphiQL is a great tool for building and verifying queries so you don't have to repeatedly rebuild your project in Xcode to try out changes. 
+
+As the schema indicates, the `launches` query returns a `LaunchConnection` object that includes both a list of launches and fields related to pagination (`cursor` and `hasMore`). The query you write indicates exactly which fields of the `LaunchConnection` object you want to be returned, like so:
+
+```graphql
+query LaunchList {
+  launches {
+    cursor
+    hasMore
+  }  
+}
+```
+
+If you run this query by pressing the play button in GraphiQL, the query returns results as a JSON object on the right-hand side of the page: 
+
+![basic query JSON in GraphiQL](images/completed_basic_query.png)
+
+This query executes successfully, but it doesn't include any information about the `launches`! That's because we didn't include the necessary field in our query.
+
+Update your query to fetch the `id` and `site` properties for each launch, like so:
+
+
+```graphql
+query LaunchList {
+  launches {
+    cursor
+    hasMore
+    launches {
+      id
+      site
+    }
+  }  
+}
+```
+
+Run the query again, and you'll now see that in addition to the information you got back before, you're also getting a list of launches with their ID and site information: 
+
+![updated query JSON in GraphiQL](images/completed_id_query.png)
+
+## Adding your query to Xcode
+
+Now that your query is fetching the right data, head back to Xcode.
+
+1. Go to **File > New > File...** and select the **Empty** file template:
+
+![](images/empty_file_template.png)
+
+2. Click **Next** and name the file `LaunchList.graphql`. Make sure it's saved at the same level as your `schema.json` file. As previously, don't add it to any target.
+
+3. Copy your final query from GraphiQL and paste it into `schema.json`. 
+
+You're now ready to generate code from the combination of your saved query and schema.
+
+## Running code generation
+
+1. Return to your project's **Apollo CLI** Run Script build phase. Comment out the line that you added to the bottom (that includes `apollo:schema`) and _uncomment_ the line you previously commented out (that includes `codegen:generate`).
```

Comment:
I think it's worth leaving the explanation of why you don't need the schema command anymore (basically, because the schema's not changing here)